frontend/src/components/pages/events/EventCreator.tsx

```diff
@@ -5,6 +5,9 @@ import { GET_USER } from "@graphql/users/queries";
 import { Category, Event } from "@interfaces/events";
 import { User } from "@interfaces/users";
 import {
+  Accordion,
```

Comment:
If you could refactor this, it would be great. Long files are difficult to handle. A good rule of thumb is that files should be shorter than 250 lines.